app/javascript/packs/notificationSubscriptionHandler.js

```diff
@@ -42,23 +42,22 @@ function loadFunctionality() {
   } else {
     updateStatus = target => {
       let payload = '';
-      const shouldUnsubscribeToNotifications = target.classList.contains('selected')
-        || target.classList.contains('selected-emoji');
+      const shouldUnsubscribeToNotifications =
+        target.classList.contains('selected') ||
+        target.classList.contains('selected-emoji');
       const allButtons = document.getElementsByClassName(
         'notification-subscription-label',
       );
       for (let i = 0; i < allButtons.length; i += 1) {
         allButtons[i].classList.remove('selected');
       }
       if (shouldUnsubscribeToNotifications) {
-        const unsubscribeButton = allButtons.namedItem(
-          'unsubscribe',
-        );
+        const unsubscribeButton = allButtons.namedItem('unsubscribe');
         unsubscribeButton.classList.add('selected');
-        payload = unsubscribeButton.dataset.payload;
+        ({ dataset: payload } = unsubscribeButton);
```

Comment:
The subscription feature is broken with this change. I believe you meant to do 
```suggestion
        ({ payload } = unsubscribeButton.dataset)
```
With that, I was able to get subscription working. Is this a new ECMAScript feature?

Reply:
Good catch, I missed this one! 😅

Reply:
Sorry about that. This was the first thing that I tried, but ESLint was giving me a hard time about it still, but now it seems to work.
Or I might have just looked at the wrong output from the linter. 😅
Anyway I made the commit with the recommended fixes.

Reply:
No worries! I'll take a look at this PR after I grab some 🍣